packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -1,6 +1,67 @@
 import React from 'react';
-import { Card } from '../../../../primitives';
+import { translate } from '@aws-amplify/ui';
+import { PreviewerProps } from '../types';
+import { Button, Card, Flex, Text, View } from '../../../../primitives';
+import { UploadDropZone } from '../UploadDropZone';
+import { UploadButton } from '../UploadButton';
+import { Tracker } from '../Tracker';
 
-export function Previewer(): JSX.Element {
-  return <Card>Previewer</Card>;
+export function Previewer({
+  acceptedFileTypes,
+  files,
+  inDropZone,
+  multiple,
+  onClose,
+  onDragEnter,
+  onDragLeave,
+  onDragOver,
+  onDragStart,
+  onDrop,
+  onFileChange,
+}: PreviewerProps): JSX.Element {
+  const CommonProps = {
+    acceptedFileTypes,
+    multiple,
+    onFileChange,
+  };
+
+  const onClick = () => {
+    // start upload
+  };
```

Comment:
I assume this is getting implemented next round of PR?

Reply:
yes